conda_build/render.py

```diff
@@ -78,7 +78,8 @@ def parse_or_try_download(metadata, no_download_source, config,
 
     need_reparse_in_env = False
     if (force_download or (not no_download_source and (metadata.uses_vcs_in_meta or
-                                                       metadata.uses_setup_py_in_meta))):
+                                                       metadata.uses_setup_py_in_meta or
```

Comment:
This is fine for now.  I'll probably refactor it later to be a collection outside of the if statement, because it's getting pretty long.  It would probably also be even nicer to refactor it so that the only changes necessary here would be in metadata.py, rather than also here.

No changes necessary on your part, just thinking here.